WordPress/src/main/res/values/dimens.xml

```diff
@@ -70,7 +70,9 @@
     <dimen name="margin_extra_extra_small">1dp</dimen>
     <dimen name="margin_extra_small">2dp</dimen>
     <dimen name="margin_small">4dp</dimen>
+    <dimen name="margin_small_medium">6dp</dimen>
     <dimen name="margin_medium">8dp</dimen>
+    <dimen name="margin_extra_medium">10dp</dimen>
```

Comment:
I was mirroring the naming of `margin_extra_large` vs `margin_large` but your suggestion makes more sense! Fixed with d99d0a3.